scripts/triggers-end-to-end-tests/run.spec.js

```diff
@@ -216,64 +212,92 @@ describe("database and firestore emulator function triggers", function () {
               },
               getCertificate: () => {
                 return {};
-              }
-            }
+              },
+            },
           });
 
           test.database_client = admin.database();
           done();
-      },
-      function (done) {
-        const firestore = test.firestore_client;
-        const database = test.database_client;
+        },
+        function(done) {
+          const firestore = test.firestore_client;
+          const database = test.database_client;
 
-        /*
+          /*
          * Install completion marker handlers and have them update state
          * in the global test fixture on success. We will later check that
          * state to determine whether the test passed or failed.
          */
-        database.ref(FIRESTORE_COMPLETION_MARKER).on("value", function (snap) {
-          test.rtdb_from_firestore = true;
-        }, function (err) {
-          assert.fail(err, "Error reading " + FIRESTORE_COMPLETION_MARKER + " from database emulator.");
-        });
-
-        database.ref(DATABASE_COMPLETION_MARKER).on("value", function (snap) {
-          test.rtdb_from_rtdb = true;
-        }, function (err) {
-          assert.fail(err, "Error reading " + DATABASE_COMPLETION_MARKER + " from database emulator.");
-        });
-
-        firestore.doc(FIRESTORE_COMPLETION_MARKER).onSnapshot(function (snap) {
-          test.firestore_from_firestore = true;
-        }, function (err) {
-          assert.fail(err, "Error reading " + FIRESTORE_COMPLETION_MARKER + " from firestore emulator.");
-        });
-
-        firestore.doc(DATABASE_COMPLETION_MARKER).onSnapshot(function (snap) {
-          test.firestore_from_rtdb = true;
-        }, function (err) {
-          assert.fail(err, "Error reading " + DATABASE_COMPLETION_MARKER + " from firestore emulator.");
-        });
-        done();
-      }
-    ], done);
+          database.ref(FIRESTORE_COMPLETION_MARKER).on(
+            "value",
+            function(/* snap */) {
```

Comment:
Why was snap commented out? Is this a new thing that formatting does for args that are not used in the function body?

Reply:
it's what _I_ did to get rid of the lint errors for unused variables. :)